### ISSUE TITLE
🐛 Bug Report: Functions(client).list return false instead of empty dictionary as response when  no variables are set

### ISSUE THREAD

**Issue by Pastajello** [appwrite/sdk-for-flutter#72]

**Comment by TorstenDittmann** [comment]
I feel like we had the issue at some point already.

@lohanidamodar do you remember this?

**Comment by lohanidamodar** [comment]
@TorstenDittmann isn't this because we are hding the VARS in client side request?

**Comment by lohanidamodar** [comment]
@Pastajello hey there's no `list` method in Flutter SDK for function service, it's only available for our https://github.com/appwrite/sdk-for-dart in the server side. Client side SDKs don't support Functions listing, please let us know the Full scenario how you are using some code sample and what SDK you are using would be great. Thanks.

**Comment by Pastajello** [comment]
@lohanidamodar Yeah, seems like I mistook the repo. This is about server side Dart SDK. :(

**Comment by lohanidamodar** [comment]
Let's move it there, also I'll test it soon and let you know.
